presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java

```diff
@@ -71,6 +71,8 @@
             "  user_regex VARCHAR(512),\n" +
             "  source_regex VARCHAR(512),\n" +
             "  client_tags VARCHAR(512),\n" +
+            "  priority BIGINT NOT NULL,\n" +
+            "  UNIQUE(priority),\n" +
```

Comment:
`UNIQUE KEY priority (priority)` is probably better.

Reply:
Do we also want to add a unique key for (user_regex, source_regex, client_tags)? With priority, you can technically have multiple entries for the same value but only the highest priority would be effective. But that might make things messy over time.

Reply:
In what ways `UNIQUE KEY` is better than `UNIQUE`?

IIUC the main difference between `UNIQUE KEY` and `UNIQUE` is that we are giving a name to the index we are creating, in this case what difference would it make to use `UNIQUE KEY priority (priority)`?

Reply:
Adding a unique key for (user_regex, source_regex, client_tags) makes sense I think.